src/Quantum/product_codes.jl

```diff
@@ -337,7 +433,7 @@ Return the pre-lifted matrices `H_X` and `H_Z` of the generalized hypergraph pro
 """
 function generalized_hypergraph_product_matrices(A::MatElem{T}, b::T) where T <: ResElem
 
-    @warn "Commutativity of A and b required but not yet enforced."
+    # @warn "Commutativity of A and b required but not yet enforced."
     S = base_ring(b)
```

Comment:
why?

Reply:
This causes warning to appear in doctests as well, with a file path to the line that causes this warning. Every user will get a different file path based on where the codingtheory is installed. 

```
julia> code = LiftedProductCode(A, b);
┌ Warning: Commutativity of A and b required but not yet enforced.
└ @ CodingTheory ~/Desktop/ct/doctests/CodingTheory/src/Quantum/product_codes.jl:340 # file path is shown with warning,this is different for each user.
```

Reply:
I think └ @ CodingTheory ~/Documents/GitHub/CodingTheory/src/Quantum/product_codes.jl:354 documenter will have this path so I can include it back.